popup.js

```diff
@@ -0,0 +1,204 @@
+const worksSection = document.getElementById('work');
+const headerWrapper = document.querySelector('.header');
+
+const works = [
+  {
+    name: 'Tonic',
+    jobDescription: {
+      company: 'CANOPY',
+      role: 'Back End Dev',
+      year: '2015',
+    },
```

Comment:
- [ ] please fix the up-down scroll in the mobile popup, make it full height so it looks like the Figma version. 
![Screenshot (162)](https://user-images.githubusercontent.com/65298048/174305986-e6f8baaa-2a43-435a-b2b0-f7ce2f843bb0.png)
figma
![Screenshot (158)](https://user-images.githubusercontent.com/65298048/174306038-75374791-66bc-46e8-8316-5fa945131f8a.png)